### ISSUE TITLE
Remove folder with one sandbox leads to crash

### ISSUE THREAD

**Issue by bfunc** [codesandbox/codesandbox-client#5637]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 89.0.4389 | Windows 10           | 602a2b11c          |

  **Route:**
  https://codesandbox.io/dashboard/all?workspace=ebce30aa-ef8b-40d2-97b3-722ff5c14875

  

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'type' of undefined
  TypeError: Cannot read property 'type' of undefined
    at Im (https://codesandbox.io/static/js/app.7b4aa0fe6.js?_sw-precache=eb87c5a329304fac8facb955565ed749:266:295341)
    at ua (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1781090)
    at Ka (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1789355)
    at xs (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1827543)
    at ml (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1811929)
    at hl (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1811854)
    at il (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1809182)
    at https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1767365
    at t.unstable_runWithPriority (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1842701)
    at qo (https://codesandbox.io/static/js/common.06f11edc9.chunk.js:1:1767074)
  
    in Im
    in Am
    in Unknown
    in t
    in t
    in div
    in Content___StyledElement
    in Unknown
    in withRouter()
    in main
    in Dashboard___StyledElement2
    in div
    in Dashboard___StyledStack2
    in div
    in Dashboard___StyledStack
    in Unknown
    in De
    in Unknown
    in yC
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in De
    in c
    in t
  ```

  </details>

**Comment by smhutch** [comment]
@bfunc are you still able to replicate this issue? I looked into it today, and was not able to replicate the error you encountered.

**Comment by lbogdan** [comment]
Looks like a duplicate of #5816 and should be fixed by #5827 .
